lib/profileAgents.js

```diff
@@ -602,6 +609,104 @@ async function _updateMeterController({meterId, controller}) {
   }
 }
 
+async function _getProfileAgentRecords({accountId, includeSecrets}) {
+  const query = {'profileAgent.account': accountId};
+  const projection = {_id: 0};
+  const collection = getCollection('profile-profileAgent');
+  let records = await collection.find(query, {projection}).toArray();
+  const zcapRefreshed = await _refreshProfileAgentZcaps({
+    profileAgentRecords: records
+  });
+  if(zcapRefreshed) {
+    records = await collection.find(query, {projection}).toArray();
```

Comment:
It's expensive to run this query again. Can we please avoid this? We should be able to get the updated results from a mongodb update call.

Reply:
No longer doing a double fetch query. Updated `profileAgents.update()` to return updated document [28ef377](https://github.com/digitalbazaar/bedrock-profile/pull/99/commits/28ef377babf23d57672538e64dc451e659c477ca)